dev-tool.sh

```
@@ -44,26 +42,64 @@ choose() {
 build() {
+  if [ -z "$ACTIVE_MODULE" ]; then
+    mvn clean install
+  else
+    mvn -pl "$ACTIVE_MODULE" -am clean install
+  fi
+}
+
+_inject_style_into_report() {
+  local html="$1"
+  local css_file="$2"
+  local css_name=$(basename "$css_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)